src/client/components/signup-google/signup-google.styles.pcss

```diff
@@ -0,0 +1,38 @@
+.container {
+  margin-top: 30px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.signup {
```

Comment:
@Mar1nka please don't use BEM notation as it's not used across the project.
Please check other components for examples of CSS classes naming